docs/Systems.md

```diff
@@ -220,35 +221,84 @@ cd rpi2_out
 make
 ```
 
-If you ever update `nerves_system_br`, be sure to run the `create-build.sh`
-script again. You can point it to the same location. Until you are comfortable
-with Buildroot, it is safest to `make clean` and then `make` to rebuild
-everything after an update to `nerves_system_br`.
+This process will take quite a while (about 30 minutes).
+When it finishes, you will have confirmed that you can successfully build the standard `rpi2` System.
+The next section will describe how to make changes and re-build the System.
+
+If you ever update `nerves_system_br`, be sure to run the `create-build.sh` script again.
+You can point it to the same location and it will update properly.
+It is best to `make clean` and then `make` to rebuild everything after updating `nerves_system_br`.
+
+### Additional Package Configuration
+
+The workflow for customizing a Nerves System is the standard Buildroot procedure using `make menuconfig`.
+The packages are divided into three categories:
+
+  1. Select base packages by running `make menuconfig`
+  2. Modify the Linux kernel and kernel modules with `make linux-menuconfig`
+  3. Enable more command line utilities using `make busybox-menuconfig`
+
+If you followed the steps in the previous section, make sure you have changed directory to `rpi2_out` first.
+
+When you quit from the `menuconfig` interface, the changes are stored temporarily.
+To save them back in your System, follow the appropriate steps below:
+
+  1. Run `make savedefconfig` after `make menuconfig` to update the `nerves_defconfig` in your System.
+  2. Run `make linux-savedefconfig` and `cp build/linux-x.y.z/defconfig <your system>` after `make linux-menuconfig`.
+     If your system doesn't contain a custom Linux configuration yet, you'll need to update the
+     Buildroot configuration to point to the new Linux defconfig in your system directory.
+     The path is usually something like `$(NERVES_DEFCONFIG_DIR)/linux-x.y_defconfig`.
+  3. For Busybox, the convention is to copy `build/busybox-x.y.z/.config` to a file in the System repository.
+     Like the Linux configuration, the Buildroot configuration will need to be updated to point to the
+     custom config.
+
+The Buildroot [user manual](http://nightly.buildroot.org/manual.html) can be very helpful especially if you need to add a package.
+The various Nerves System repositories have examples of many common use cases, so check them out as well.
+
+### How to Use Your New System
+
+By default, Nerves downloads pre-built Systems from the Internet and caches them on your hard drive.
+To force a local build and not cache (so you can re-build it), set the following environment variables:
+
+```
+NERVES_SYSTEM_CACHE=none
+NERVES_SYSTEM_COMPILER=local
```

Comment:
I never do this step. I only update `NERVES_SYSTEM`. @mobileoverlord - am I doing it right?